tests/acceptance/tasks/ShopAdmin/Category/CreateLandingPage.ts

```diff
@@ -27,9 +27,20 @@ export const CreateLandingPage = base.extend<{ CreateLandingPage: Task }, Fixtur
                     // Select existing layout
                     await AdminLandingPageCreate.assignLayoutButton.click();
                     // Search input need to delay press more than 300ms to mimic user typing in order to activate search action
-                    await AdminLandingPageCreate.searchLayoutInput.pressSequentially(layoutName.substring(0,5), {delay: 500});
+                    await AdminLandingPageCreate.searchLayoutInput.pressSequentially(layoutName.substring(0, 5), { delay: 500 });
+
+                    const gridLocator = AdminLandingPageCreate.page.locator('.sw-data-grid__cell-content').first();
+                    const gridVisiable = await gridLocator.isVisible();
+                    if (gridVisiable) {
+                        await AdminLandingPageCreate.page.getByLabel('Select layout').locator('div').filter({ hasText: 'Sort by:' }).getByRole('button').first().click();
+                    }
                     await AdminLandingPageCreate.page.getByTitle(layoutName).click();
-                    await AdminLandingPageCreate.layoutSaveButton.click();
+
+                    if (gridVisiable) {
+                        await AdminLandingPageCreate.page.getByRole('button', { name: 'Add', exact: true }).click();
+                    } else {
+                        await AdminLandingPageCreate.layoutSaveButton.click();
+                    }
```

Comment:
This is pretty hacky. For some reason the grid view default is different on saas